### ISSUE TITLE
 A newer version does not work correctly

### ISSUE THREAD

**Issue by BarGol00683** [stephanrauh/ngx-extended-pdf-viewer#97]
My package.json was still updating your lib for newest versions. I started developing my app from 0.9.46 lib version.
0.9.48 - 0.9.51 created a few problems for me:
- pdf displayed like a picture
- CTRL+S or downloadButton still downloading undefined.pdf even with property `[filenameForDownload]`
- searching through text didn't work ( CTRL + F )

I didn't manage to find other problems.

.ts
creating object from blob
```typescript
public file: string | ArrayBuffer = null;

getFile(_fileId: number) {
    this.someService.getFileById(_fileId).subscribe(
      data => {
        const blob = new Blob([data], {type: 'application/pdf'});
        this.file = URL.createObjectURL(blob);
      }
    )
  }
```

.html

```html
<ngx-extended-pdf-viewer [src]="file"
                            useBrowserLocale="true"
                            delayFirstView="1000"
                            height='100%'
                            [filenameForDownload]="fileName"
                            [showPrintButton]="false">
    </ngx-extended-pdf-viewer>
```

So after a couple of hours i downloaded the older version and it was my sollution.

**Comment by stephanrauh** [comment]
- I've fixed the `filenameForDownload` bit (see #96). 
 - The other two bugs are probably not real bugs but an optimization. Please add the option `[showHandTool]="true"`. That should solve both issues.

Please report back if that helps you - and more importantly, if it doesn't. I'll close the ticket until I hear from you.

Thanks in advance,
Stephan

**Comment by BarGol00683** [comment]
@stephanrauh 
Now with 0.9.52 works fine.
